mirage/serializers/resource.js

```diff
@@ -1,6 +1,11 @@
 import ApplicationSerializer from './application';
 
 function mapResourceAttrs(hash, resource) {
+  // if the ID is 1, it'll be a custom title
+  if (resource.id === '1') {
```

Comment:
It's still hard-coded to that ID :( 

What if you did `hash.attributes.isTitleCustom = resource.title.isTitleCustom;` here?

Reply:
Ah, how would we knowingly get to a custom title / resource? Would it be with a query param?

Reply:
Ah, I'll just remove it then.